Google.PowerShell/Compute/GceMetadataCmdlet.cs

```diff
@@ -115,8 +117,10 @@ public class GetGceMetadataCmdlet : GCloudCmdlet
 
         /// <summary>
         /// Make this a field, so it can be aborted if cmdlet stops e.g. the user hits Ctrl-C.
+        /// Many properties of HttpWebRequest are not available on CoreCLR so it is better
```

Comment:
Mentioning HttpWebRequest doesn't make sense, since the future version of the code will only have HttpClient. Right? So I'd just drop these comments, but keep your refactorings.